File: app/src/main/java/com/aaron/yespdf/main/OperationBarHelper.java
package com.aaron.yespdf.main;

import android.view.View;

/**
 * @author Aaron aaronzzxup@gmail.com
 */
final class OperationBarHelper {

    static void show(View view) {
        view.setVisibility(View.VISIBLE);
//        view.animate()
//                .setDuration(200)
//                .alpha(1)
//                .setListener(new AnimatorListenerAdapter() {
//                    @Override
//                    public void onAnimationStart(Animator animation) {
//                        view.setVisibility(View.VISIBLE);
//                    }
//                })
//                .start();
    }

    static void hide(View view) {
        view.setVisibility(View.GONE);
//        view.animate()
//                .setDuration(200)
//                .alpha(0)
//                .setListener(new AnimatorListenerAdapter() {
//                    @Override
//                    public void onAnimationEnd(Animator animation) {
//                        view.setVisibility(View.GONE);
//                    }
//                })
//                .start();
    }

    private OperationBarHelper() {
    }
}

File: app/src/main/java/com/aaron/yespdf/main/OperationBarHelper.kt
package com.aaron.yespdf.main

import android.animation.Animator
import android.animation.AnimatorListenerAdapter
import android.view.View
import android.view.animation.AccelerateDecelerateInterpolator
import android.view.animation.BounceInterpolator

/**
 * @author Aaron aaronzzxup@gmail.com
 */
object OperationBarHelper {
    fun show(view: View) {
        view.animate()
                .scaleX(1.0F)
                .scaleY(1.0F)
                .alpha(1.0F)
                .setDuration(100L)
                .setInterpolator(AccelerateDecelerateInterpolator())
                .setListener(object : AnimatorListenerAdapter() {
                    override fun onAnimationStart(animation: Animator?) {
                        view.visibility = View.VISIBLE
                    }
                })
                .start()
    }

    fun hide(view: View) {
        view.animate()
                .scaleX(1.1F)
                .scaleY(1.1F)
                .alpha(0F)
                .setDuration(100L)
                .setInterpolator(AccelerateDecelerateInterpolator())
                .setListener(object : AnimatorListenerAdapter() {
                    override fun onAnimationEnd(animation: Animator?) {
                        view.visibility = View.GONE
                    }
                })
                .start()
    }
}